app/workers/publishing_api_notifier.rb

```diff
@@ -5,8 +5,8 @@ def perform(edition_id, update_type = "normal")
     edition = Edition.find(edition_id)
```

Comment:
Right, could it be "PublishingApiPublishNotifier" to be similar to the other one?